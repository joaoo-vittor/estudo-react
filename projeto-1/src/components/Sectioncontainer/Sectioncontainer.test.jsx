import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/renderTheme';
import { Sectioncontainer } from '.';

describe('<Sectioncontainer />', () => {
  it('should render content', () => {
    const { container } = renderTheme(
      <Sectioncontainer>
        <h1>Children</h1>
      </Sectioncontainer>,
    );
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
