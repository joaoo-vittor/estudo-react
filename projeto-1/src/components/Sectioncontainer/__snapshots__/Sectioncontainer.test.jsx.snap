// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sectioncontainer /> should render content 1`] = `
.c0 {
  max-width: 120rem;
  margin: 0;
  padding: 3.2rem;
}

<div>
  <div
    class="c0"
  >
    <h1>
      Children
    </h1>
  </div>
</div>
`;
